sources/packages/api/src/types/institutionLocation.ts

```diff
@@ -0,0 +1,26 @@
+export interface InstitutionLocationType {
+  address1: string;
+  address2?: string;
+  applicationId: string;
+  city: string;
+  country: string;
+  locationName: string;
+  postalZipCode: string;
+  provinceState: string;
+  submit: boolean;
```

Comment:
Please consider removing the submit from here since it is not part of the business model.